packages/instantsearch.js/src/lib/insights/__tests__/insights-listener-test.tsx

```diff
@@ -128,7 +130,7 @@ describe('createInsightsEventHandler', () => {
       const payload = serializePayload(
         _buildEventPayloadsForHits({
           widgetType: 'ais.hits',
-          getIndex: () => 'instant_search',
+          helper: algoliasearchHelper({} as SearchClient, 'instant_search'),
```

Comment:
doesn't matter much, but you can also use the `createSearchClient` from `@instantsearch/testutils`

Reply:
ha, I couldn't find it!
I'll update the test 🙂